geometry/Goal.cpp

```diff
@@ -37,14 +37,12 @@ Goal::Goal()
      _isFinalGoal=0;
      _walls = std::vector<Wall > ();
      _poly = std::vector<Point > ();
-     _crossing = new Crossing();
      _roomID = -1;
      _subRoomID = -1;
 }
 
 Goal::~Goal()
 {
-
```

Comment:
What about using `~Goal() = default` in header?

Reply:
Changed, thanks.